WebDevStudios/wds-bt
# feature / WDSBT-10 remove hardcoded icons

# Closes

https://webdevstudios.atlassian.net/browse/WDSBT-10 (PARTIALLY)

## Link to test

coming soon...

## Description

Remove the hardcoded arrow icon on buttons. Instead of hardcoding the arrow icon, users can use an icon plugin to have more control.

## Related Tickets & Documents

https://github.com/WebDevStudios/wds-icons

## Mobile & Desktop Screenshots/Recordings

N/A, icons were removed from the buttons.

## Added to documentation?


## Added tests?


## Testing Instructions

1. Pull this branch, verify that the hardcoded icons are gone, verify the button's padding looks good for all button sizes.

-----

## Reviewer's Testing Checklist



As a reviewer, please verify that the relevant testing criteria are fulfilled and confirmed before approving this Pull Request.


## [optional] Additional Reviewer Notes or Considerations?
